### PR TITLE
Make $project.Type return "C#" for new CPS-based C# projects

### PR DESCRIPTION
**Pull request by bricelam**
Guid comes from [here](https://github.com/dotnet/roslyn-project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/Packaging/CSharpProjectSystemPackage.cs#L36).

### CODE REVIEW

**Comment by rrelyea** [comment]
@bricelam - is there an issue filed that shows the problem if we don't fix this.
Need to decide which milestone we take the fix. If we are trying to take now, need to understand what is broken if we don't.

**Comment by bricelam** [comment]
Not urgent. I just noticed it was missing. It was more efficient to submit a PR than file a bug.

File: src/NuGet.Clients/VsConsole/PowerShellHost/Scripts/NuGet.Types.ps1xml [hunk 1]
```diff
@@ -61,6 +61,7 @@
                 <GetScriptBlock>
                     switch ($this.Kind) {
                         "{262852C6-CD72-467D-83FE-5EEB1973A190}" {"JavaScript"}
+                        "{9A19103F-16F7-4668-BE54-9A1E7A4F7556}" {"C#"}
```
Comment (yishaigalatzer) [inline src/NuGet.Clients/VsConsole/PowerShellHost/Scripts/NuGet.Types.ps1xml:64]: If this is the case, we should probably fix F#, VB and perhaps typescript? along the way

**Comment by bricelam** [comment]
Error 408: Pull request timeout.

**Pull request closed without merging**